### PR TITLE
plone 6

### PR DESCRIPTION
**Pull request by mamico**

### CODE REVIEW

**Review by ale-rt — APPROVED** [review]
:+1: I am fine merging this and releasing even if the package does not officially declares support for Plone6, we can do that in another PR

**Comment by mamico** [comment]
@ale-rt thx

**Review by ale-rt — APPROVED** [review]

File: CHANGES.rst [hunk 1]
```diff
@@ -4,8 +4,8 @@ Changes log
 1.2.7 (unreleased)
 ------------------
 
-- Nothing changed yet.
-
+- Add ++webresource++ to valid_subpart_prefixes (Plone 6)
```
Comment (ale-rt) [inline CHANGES.rst:7]:
```suggestion
- Add ++webresource++ and ++plone++ to valid_subpart_prefixes (Plone 6)
```

File: iw/rejectanonymous/__init__.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ class IPrivateSite(Interface):
 ))
 
 valid_subpart_prefixes = frozenset(('++resource++', '++theme++', '++plone++static',
-    '++unique++',
+    '++unique++', '++webresource++', '++plone++',
```
Comment (ale-rt) [inline iw/rejectanonymous/__init__.py:52]: Should the `++plone++static`  prefix be removed from the line above?

**Review by ale-rt — APPROVED** [review]

**Comment by ale-rt** [comment]
Released: https://pypi.org/project/iw.rejectanonymous/1.2.7/

**Pull request merged**